pkg/systemd/services/service-details.jsx

```diff
@@ -327,6 +379,9 @@ export class ServiceDetails extends React.Component {
         const masked = this.props.unit.LoadState === "masked";
         const unit = this.state.unit_properties;
         const showAction = this.props.permitted || this.props.owner == "user";
+        // These timers are added by cockpit via the UI
+        const isCustom = this.props.unit.FragmentPath.startsWith("/etc/systemd/system") && !masked;
```

Comment:
These are not cockpit specific. Any custom timer should be placed there. It's fine if cockpit can delete them, but we shouldn't pretend that they were created by Cockpit (not unless we add some `X-Created-By: Cockpit` to the unit or similar -- but IMHO we shouldn't).